code/weapon/weapons.cpp

```diff
@@ -6792,7 +6792,7 @@ void weapon_hit_do_sound(object *hit_obj, weapon_info *wip, vec3d *hitpos, bool
 		// play a shield hit if shields are above 10% max in this quadrant
 		if ( shield_str > 0.1f ) {
 			// Play a shield impact sound effect
-			if ( hit_obj == Player_obj ) {
+			if ( !(Use_weapon_class_sounds_for_hits_to_player) && (hit_obj == Player_obj)) {
```

Comment:
What will happen if the weapon impact sound is not specified?

Reply:
As far as I know that's just one of those fields that needs to be specified in the weapons table

Reply:
That makes sense.  Wouldn't want a weapon to silently impact.

Reply:
Well nothing *needs* to be specified now that we have TBMs...